crates/re_log_types/src/lib.rs

```diff
@@ -205,6 +205,27 @@ impl std::fmt::Display for ApplicationId {
 
 // ----------------------------------------------------------------------------
 
+/// Options for configuring viewer behavior when a blueprint is ready.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)] // `PartialEq` used for tests in another crate
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct BlueprintReadyOpts {
+    /// Make this the active blueprint for the `app_id`.
+    pub make_active: bool,
+
+    /// Make this the default blueprint for the `app_id`.
+    pub make_default: bool,
+}
+
+impl std::fmt::Display for BlueprintReadyOpts {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "BlueprintReadyOpts {{ make_active: {}, make_default: {} }}",
+            self.make_active, self.make_default
+        )
+    }
+}
```

Comment:
This looks like the wrong use of `Display` to me. This is more the thing you want from `Debug`.

If something implements `Display` it should be nice enough to show in a UI as is, which I don't think this is.